libs/core/src/components/pds-dropdown-menu/docs/pds-dropdown-menu.mdx

```diff
@@ -0,0 +1,250 @@
+
+### Default
+
+The default dropdown menu displays a list of options that can be selected by the user.
+
+<DocCanvas client:only
+  mdxSource={{
+    react: `
+<PdsDropdown>
+  <PdsButton slot="trigger">Trigger</PdsButton>
+  <PdsDropdownItem>Item 1</PdsDropdownItem>
+  <PdsDropdownItem>Item 2</PdsDropdownItem>
+  <PdsDropdownItem>Item 3</PdsDropdownItem>
+</PdsDropdown>
+`,
+    webComponent:`
+<pds-dropdown>
+  <pds-button slot="trigger">Trigger</pds-button>
+  <pds-dropdown-item>Item 1</pds-dropdown-item>
+  <pds-dropdown-item>Item 2</pds-dropdown-item>
+  <pds-dropdown-item>Item 3</pds-dropdown-item>
+</pds-dropdown>
+`
+  }}
+>
+<div style={{ height: '200px', width: '100%', textAlign: 'center'}}>
+  <pds-dropdown-menu>
+    <pds-button slot="trigger">Trigger</pds-button>
+    <pds-dropdown-menu-item>Item 1</pds-dropdown-menu-item>
+    <pds-dropdown-menu-item>Item 2</pds-dropdown-menu-item>
+    <pds-dropdown-menu-item>Item 3</pds-dropdown-menu-item>
+  </pds-dropdown-menu>
+</div>
+</DocCanvas>
+
+### With Separator
+
+The dropdown menu can also display a separator between items.
+
+<DocCanvas client:only
+  mdxSource={{
+    react: `
+<PdsDropdownMenu>
+  <PdsButton slot="trigger">Trigger</PdsButton>
+  <PdsDropdownMenuItem>Item 1</PdsDropdownMenuItem>
+  <PdsDropdownMenuSeparator />
+  <PdsDropdownMenuItem>Item 2</PdsDropdownMenuItem>
+  <PdsDropdownMenuSeparator />
+  <PdsDropdownMenuItem>Item 3</PdsDropdownMenuItem>
+</PdsDropdownMenu>
+`,
+    webComponent:`
+<pds-dropdown-menu>
+  <pds-button slot="trigger">Trigger</pds-button>
+  <pds-dropdown-menu-item>Item 1</pds-dropdown-menu-item>
+  <pds-dropdown-menu-separator />
+  <pds-dropdown-menu-item>Item 2</pds-dropdown-menu-item>
+  <pds-dropdown-menu-separator />
+  <pds-dropdown-menu-item>Item 3</pds-dropdown-menu-item>
+</pds-dropdown-menu>
+`
+  }}
+>
+<div style={{ height: '200px', width: '100%', textAlign: 'center' }}>
+  <pds-dropdown-menu>
+    <pds-button slot="trigger">Trigger</pds-button>
+    <pds-dropdown-menu-item>Item 1</pds-dropdown-menu-item>
+    <pds-dropdown-menu-separator />
+    <pds-dropdown-menu-item>Item 2</pds-dropdown-menu-item>
+  </pds-dropdown-menu>
+</div>
+</DocCanvas>
+
+### With Links
+
+Menu items can function as links by using the href attribute.
+
+<DocCanvas client:only
+  mdxSource={{
+    react: `
+<PdsDropdownMenu>
+  <PdsButton slot="trigger">Navigation</PdsButton>
+  <PdsDropdownMenuItem href="/dashboard">Dashboard</PdsDropdownMenuItem>
+  <PdsDropdownMenuItem href="/profile">Profile</PdsDropdownMenuItem>
+  <PdsDropdownMenuItem href="/settings">Settings</PdsDropdownMenuItem>
+</PdsDropdownMenu>
+`,
+    webComponent:`
+<pds-dropdown-menu>
+  <pds-button slot="trigger">Navigation</pds-button>
+  <pds-dropdown-menu-item href="/dashboard">Dashboard</pds-dropdown-menu-item>
+  <pds-dropdown-menu-item href="/profile">Profile</pds-dropdown-menu-item>
+  <pds-dropdown-menu-item href="/settings">Settings</pds-dropdown-menu-item>
```

Comment:
The hover effect is triggered when hovering anywhere over a `menu-item`, but I can only click a link if I have the mouse over the text. We should have it so we can click anywhere on the menu item as well, if that's how the hover behaves. The button variant seems fine.

You can see in the screenshot that the link doesn't expand to fill the menu-item area:

![Screenshot 2025-06-02 at 8 40 47 AM](https://github.com/user-attachments/assets/13103771-db7e-4a1e-89ed-06e6a1372f37)